src/modules/ekf2/EKF/mag_heading_control.cpp

```diff
@@ -68,18 +68,34 @@ void Ekf::controlMagHeadingFusion(const magSample &mag_sample, const bool common
 	if (_control_status.flags.yaw_align) {
 		// mag heading
 		aid_src.innovation = wrap_pi(getEulerYaw(_R_to_earth) - aid_src.observation);
+		_mag_heading_innov_lpf.update(aid_src.innovation);
+
+		if (!_control_status.flags.gps) {
+			_control_status.flags.mag_consistent = true;
+
+		} else if (fabsf(_mag_heading_innov_lpf.getState()) < math::radians(10.f)) {
```

Comment:
Anything noteworthy about 10 degrees?

Reply:
Changed it to use a std_dev of mag heading noise param